.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,6 +17,13 @@ items that may help during the review.-->
 - [ ] Covered the changes with automated tests
 - [ ] Tested the changes locally
 
+### Cloud Review Checklist
+
+Once the Jenkins artifact is ready, run a local cloud instance and:
```

Comment:
Can this link to instructions on running a cloud instance locally? Also, don't you need to wait for ent-search to build a cloud instance you can run locally? I thought that was part of the ent-search build, not part of the connectors-python build.